extensions/azurecore/package.json

```diff
@@ -169,6 +169,7 @@
 		"@types/request": "^2.48.1",
 		"mocha": "^5.2.0",
 		"should": "^13.2.1",
-		"typemoq": "^2.1.0"
+		"typemoq": "^2.1.0",
+		"vscode": "^1.1.33"
```

Comment:
You shouldn't need to bring this in here. we ref the .d.ts directly https://github.com/Microsoft/azuredatastudio/blob/master/extensions/azurecore/src/typings/ref.d.ts#L6

Reply:
I need this for the bin folder - testrunner is referenced from `const testRunner = require('vscode/lib/testrunner');` in the main index.ts. Without this, integration tests were broken.

Incidentally we might want to look at an alternate way for our built-in extensions to reference and use this, but this is how a few others already work.